templates/home.html

```diff
@@ -18,149 +35,152 @@
-            <a href="/login"><b>Log in</b></a>/<a href="/register"><b>Register</b></a> to create a new private lobby and play with your friends!
-        </p>
+        <p>Try our new prompt generator <a href="/generator">here</a>!</p>
     </div>
 
     <hr class="my-4">
 
-    <div class="row justify-content-md-center py-4">
-        <div class="col-sm-6">
-            <div class="card">
-                <div class="alert alert-info my-0" role="alert">
-                    <h5 class="my-0">Prompt of the Day</h5>
-                </div>
-                <div class="card-body">
-                    <template v-if="dailyPrompts.length > 0">
-                        <h6 >Time Remaining: [[timeLeft]]</h6>
-
-                        <table class="table table-hover">
-                            <thead>
-                                <th scope="col">Prompt #</th>
-                                <th scope="col">Starting Article</th>
-                                <th class="text-center"></th>
-                            </thead>
-                            <tbody>
-                                <tr v-for="prompt in dailyPrompts" v-cloak>
-                                    <td>[[prompt.prompt_id]]</span><span v-if="loggedIn"> (<a v-bind:href="'/play/' + prompt.prompt_id">play</a>)</span></td>
-
-                                    <td>[[prompt.start]]</td>
-
-                                    <td class="text-center">
-                                        <template v-if="prompt.played">
-                                            <a v-bind:href="'/leaderboard/' + prompt.prompt_id">Results</a>
-                                        </template>
-                                        <template v-else>
-                                            <span title="Play this prompt to see the results!">-</span>
-                                        </template>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </template>
-                    <template v-else>
-                        <p>Currently unavailable</p>
-                    </template>
-                </div>
-                <div v-cloak class="card-footer text-muted" v-if="loggedIn === true">
-                    Play the daily prompt to start/continue your streak! 🔥
-                </div>
-                <div v-cloak class="card-footer text-muted" v-else>
-                    Log in to play this daily prompt and start your streak! 🔥
-                </div>
+    <!-- sidebar, which will move to the top on a small screen -->
+    <div class="col-sm-3">
+        <nav id="toc" data-toggle="toc" class="sticky-top"></nav>
+    </div>
+    <!-- main content area -->
+    <div class="col-sm-9">
+        <div class="card">
+            <div class="alert alert-info my-0" role="alert">
+                <h5 class="my-0">Prompt of the day</h5>
```

Comment:
nit: I think "Prompt of the Day" would be better as this is a [title](https://capitalizemytitle.com/)